hack/gen-swagger-doc/deploy.sh

```diff
@@ -5,24 +5,44 @@ set -e
 GITHUB_FQDN=github.com
 API_REF_REPO=kubevirt-incubator/api-reference
 API_REF_DIR=/tmp/api-reference
+GITHUB_IO_FQDN="https://kubevirt-incubator.github.io/api-reference"
+
+TARGET_DIR="$TRAVIS_BRANCH"
+if [ -n "${TRAVIS_TAG}" ] ; then
+    TARGET_DIR="$TRAVIS_TAG"
+fi
 
 git clone \
     "https://${API_REFERENCE_PUSH_TOKEN}@${GITHUB_FQDN}/${API_REF_REPO}.git" \
     "${API_REF_DIR}" >/dev/null 2>&1
-rm -rf "${API_REF_DIR}/content/"*
-cp -f _out/apidocs/html/*.html "${API_REF_DIR}/content/"
+rm -rf "${API_REF_DIR}/${TARGET_DIR:?}/"*
+cp -f _out/apidocs/html/*.html "${API_REF_DIR}/${TARGET_DIR}/"
 
 cd "${API_REF_DIR}"
 
+# Generate README.md file
+cat > README.md << __EOF__
+# KubeVirt API Reference
+
+Content of this repository is generated from OpenAPI specification of
+[KubeVirt project](https://github.com/kubevirt/kubevirt) .
+
+## KubeVirt API References
+
+* [master](${GITHUB_IO_FQDN}/index.html)
```

Comment:
Shouldn't the master documentation be in a `master` sub-folder because of `$TARGET_DIR`?

Reply:
I did rebase on latest changes and resolved conflicts.
Yes you are right, there is missing `master` sub directory, thanks.